File: 76/76.rb
def factorial(num) 
	return 1 if num == 0
	return num*factorial(num-1)
end

def c(x,y)
	return factorial(x)/(factorial(y)*factorial(x-y))
end


# ans = 0
# (1..99).each do |i|
# 	ans += (c(100,i)/(1.0*c(99,i))).floor
# end

# puts ans

File: 76/76.js
var ans = 0;

// maxPart is the largest integer in the sum, it also MUST be present(This is where I have been messing up)
function sumTo(num, maxPart) {

	var remainder = num - maxPart;
	if(remainder <= maxPart) ans++;
	if(maxPart <= 0) return;

	if(remainder > maxPart) 
		sumTo(remainder, maxPart);
	else
		sumTo(remainder, remainder - 1);

	sumTo(num, maxPart - 1);
}


var goal = 100;
sumTo(goal, goal-1);

console.log(ans);